csharp/ql/test/query-tests/API Abuse/FormatInvalid/FormatMissingArgument.cs

```diff
@@ -8,27 +8,27 @@ void TestFormatMissingArgument()
         String.Format("{0}", 0);
 
         // BAD: Missing {1}
-        String.Format("{1}", 0);
+        String.Format("{1}", 0); // $ Alert Sink
 
         // BAD: Missing {2} and {3}
-        String.Format("{2} {3}", 0, 1);
+        String.Format("{2} {3}", 0, 1); // $ Alert Sink
 
         // GOOD: An array has been supplied.
         String.Format("{0} {1} {2}", args);
 
         // GOOD: All arguments supplied to params
         String.Format("{0} {1} {2} {3}", 0, 1, 2, 3);
 
-        helper("{1}");
+        helper("{1}"); // $ Source
```

Comment:
I think this should be e.g. `Source=source1` and then `$ Alert=source1` further down.